docs/introduction.org

```diff
@@ -0,0 +1,106 @@
+* Introduction
+
+Nowadays, multiple languages have emerged to compete with the well-established C language.
+In fact, C is widely used in many places, learned by a lot of engineers, and used for many purposes.
+C is a powerful tool, /but with great power comes great responsibility/, in fact, memory leaks can easily occur by mistake.
+
+That's why new languages come to find a better way to handle errors and memory.
+The Rust language relies strongly on its borrow checker to avoid safe memory but it comes at a cost: increased code complexity
```

Comment:
is code complexity really the only cost ?

Reply:
Is now more clear ?

Reply:
I meant that it could not only be code complexity but code performance aswell ?

Reply:
For the perfs, the difference will depend a lot of the opti used for the program. It's not like a comparison with C and python

Reply:
Benchmark example: [zig-vs-rust](https://programming-language-benchmarks.vercel.app/zig-vs-rust)

Reply:
From what I have read, I don't think it decreases the performances: https://doc.rust-lang.org/book/ch04-01-what-is-ownership.html